styles.css

```diff
@@ -320,3 +367,50 @@ header {
 .portfolio-details h4 {
   font-size: 24px;
 }
+
+.inputs {
+  display: flex;
+  flex-direction: column;
+}
+
+.contacr_form {
+  display: flex;
+  flex-direction: column;
+}
+
+.contact .form1 {
+  padding: 12px 67px 12px 0;
+  margin-top: 50px;
+  outline: none;
+}
+
+.contact .form2 {
+  padding: 12px 67px 12px 0;
+  margin-top: 20px;
+  outline: none;
+}
+
+.contact .text_form {
+  /* padding: 10px 67px 77px 0; */
+  padding: 10px 35px 77px 0;
+  margin-top: 20px;
+  border: solid 1px green;
+  outline: none;
+  border-radius: 4px;
+}
```

Comment:
You have done very well on the contact form and its functionality 👍 . However, there seems to be a bit of a deviation from the Figma design template with regard to the sizing of the ```text area```. I suggest that you consider a width value for the element and get it to be centered in the middle. for instance, you could add these properties to the rule: 

```sh
   margin-left: auto;
   margin-right: auto;
```

Please check the screenshot below: 

Your design: 
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/107795045/5168a6d8-f381-4312-8cfb-510efe853df0)

Figma design: 
![image](https://github.com/AbdulSalamAbro/portfolio-website/assets/107795045/bac1f7a4-8295-4137-9423-84d716ae837c)